### ISSUE TITLE
Bad format for data.INDIVIDUAL(1).RESPONSE.NAME

### ISSUE THREAD

**Issue by WJVangie** [MarcLavielle/mlxR#9]
Hello Marc,
I want to generate data of a piecewise growth model whith two diffirent group.
**Here is my R code on RStudio:**
`library(Rcpp)
library(ggplot2)
library(reshape)
library(mlxR)
#----------------------------------------------------

#--------------------------------------------------------------------------
N = 100
n1 <- 0.75*N 
n2 <- 0.25*N
f <- list(time=seq(0, 6, by=1)) 
p1 <- c(T=3, a_i=0, b_i=1, d_i=0.6, D=0.2, D_a=1, D_b=0.5, D_d=0.2)
p2 <- c(T=3, a_i=1, b_i=1.73, d_i=0.8, D=0.2, D_a=1, D_b=0.5, D_d=0.2)
y <- list(name='y')
a <- list(name='a')
b <- list(name='b')
d <- list(name='d')
out <- list(a, b, d, y)
g1 <- list( paremeter = p1, size  = n1,  level ='longitudina')
g2 <- list( paremeter = p2, size  = n2,  level ='longitudina')
res <- simulx(model      = pgmm.model,
              output     = out,
               group     = list(g1,g2),
               settings  = list(seed = 12345))`

When I run the code, there is an error:
 **Error in simulxunit(model = model, lv = lv, settings = settings, out.trt = out.trt) : 
  Bad format for data.INDIVIDUAL(1).RESPONSE.NAME 
	 Shall contain vector of string**

I really don't know what is wrong with my code.Could you please help me?
Many thanks
Vangie

**Comment by MarcLavielle** [comment]
Hello,

time is missing in the output y

Marc

2016-04-20 4:27 GMT+02:00 WJVangie <notifications@github.com>:

> Hello Marc,
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MarcLavielle/mlxR/issues/9>
>



-- 
<http://www.cmap.polytechnique.fr/~lavielle/book.html>

*Marc Lavielle*

*CMAP, Ecole Polytechnique91128 Palaiseau Cedex *

*France*
Tel: (33) 1 69 33 46 47
       (33) 6 37 31 93 05

Marc.Lavielle@inria.fr
http://www.cmap.polytechnique.fr/~lavielle

"""""""""""""""""""""""""""""""""""""""""""""""""""""""""

**Comment by WJVangie** [comment]
Dear Marc,
Thank you very much!
 I added time  in the output y, changing the code`y <- list(name='y') ` into a new one `y <- list(name='y',time=seq(0, 6, by=1))`, then run the whole R code.
 A new ERROR appears:
**Error in simulxunit(model = model, lv = lv, settings = settings, out.trt = out.trt) : 
  Unexpected field in 'DATA' input structure
	 data.GROUP(1).PAREMETER found is unknown.
	 Please use only expected field names described in documentation**
I checked the parameter in model, and changed the name of them.It didn't work. What does the error mean?
